lib/aca_entities/payment_transactions/transaction.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module PaymentTransactions
```

Comment:
Also, here to AcaEntities::PaymentTransaction.

Reply:
Removed payment transactions namespace.